src/main/java/bh/bot/app/AbstractApplication.java

```diff
@@ -1315,6 +1327,117 @@ protected boolean tryEnterWorldBoss(boolean doWorldBoss, UserConfig userConfig,
 		return clickImage(BwMatrixMeta.Metas.WorldBoss.Buttons.summonOnListingWorldBosses);
 	}
 
+	protected boolean tryEnterQuest(boolean doQuest, UserConfig userConfig, Supplier<Boolean> isBlocked,
```

Comment:
Quick question, the entire change is all about this function?

Reply:
Was yes, now it also adds a small task to make sure the leave dungeon window gets exited